src/components/CytoscapeLayout/graphs/GraphStyles.ts

```diff
@@ -41,6 +41,12 @@ export class GraphStyles {
           'border-color': '#0088ce' // pf-blue
         }
       },
+      {
+        selector: 'node[isRoot]',
+        style: {
+          shape: 'diamond'
```

Comment:
this introduces a different shape for a node? I'm not convinced I should be the one saying OK to this.